AsyncDisplayKit/Layout/ASCenterLayoutSpec.mm

```diff
@@ -17,6 +17,7 @@ @implementation ASCenterLayoutSpec
 {
   ASCenterLayoutSpecCenteringOptions _centeringOptions;
   ASCenterLayoutSpecSizingOptions _sizingOptions;
+  ASRelativeLayoutSpec *_internalLayoutSpec;
```

Comment:
I suspect this can't be done because of the difference in property types while using the same names, but...it would be ideal if Center could be a subclass of Relative.  This would appreciably reduce object allocation cost / method forwarding, although I am not worried about it only because Center is rare to use extensively and we hope more people will start using Relative directly :).